recipe/meta.yaml

```
@@ -1,90 +1,121 @@
 {% set name = "glue-core" %}
-{% set version = "1.2.4" %}
+{% set version = "1.22.2" %}
 
 package:
-  name: {{ name }}
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: glue-core-{{version}}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: e8747b5a4f98241d707a5d6ba7498033d0c450130cc1f6074a0cc9ef6de12853
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name.replace('-', '_')}}-{{ version }}.tar.gz
+  sha256: 2c177daea8b76b62ed9569d146cb8b10bbaaee824944355845270ea6ac1e7b29
 
 build:
   number: 0
-  skip: True  # [ppc64le or py<37]
-  script: python -m pip install --no-deps --ignore-installed .
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
-    - glue = glue.main:main
     - glue-config = glue.config_gen:main
-    - glue-deps = glue._deps:main
-  osx_is_app: True
-  ignore_run_exports:
-    - pyqt
+  skip: True  # [py<310]
 
 requirements:
   host:
     - python
     - pip
-    # We need to include PyQt5 here to make sure that PyQt5 doesn't
-    # get included in install_requires
-    - pyqt >=5.9
     - setuptools_scm
     - setuptools >=30.3.0
     - wheel
   run:
-    # The following is needed to make sure that the package works as a GUI
-    # application (glue needs to be run with python.app, not python)
-    - python.app  # [osx]
-
-    # Required dependencies
     - python
     - numpy >=1.17
-    - matplotlib-base >=3.2
+    - matplotlib >=3.2
     - scipy >=1.1
     - pandas >=1.2
-    - echo >=0.5
+    - echo >=0.6
     - astropy >=4.0
-    - setuptools >=30.3.0
-    - qtpy >=1.9
+    - fast-histogram >=0.12
     - ipython >=4.0
-    - ipykernel >=4.0,!=5.0.0,!=5.1.0
-    - qtconsole >=4.3
     - dill >=0.2
-    - xlrd >=1.2
     - h5py >=2.10
-    - mpl-scatter-density >=0.7
+    - xlrd >=1.2
     - openpyxl >=3.0
-
-    # Optional dependencies (defined in ``extras_require``)
-    - scikit-image
+    - mpl-scatter-density >=0.8
+    - importlib_resources >=1.3  # [py<39]
+    - importlib_metadata >=3.6  # [py<310]
+    - shapely >=2.0
```

```suggestion
    - shapely >=2.0
    # recommended
    - scikit-image
```
scikit-image is listed as recommended, and the project maintainer listed it in the cf recipe.